Packages/Style/Sources/Images.swift

```diff
@@ -196,6 +196,10 @@ public enum Images {
         public static let warning = Image(.verificationOrange)
         public static let risk = Image(.verificationRed)
     }
+    
+    public enum Banners {
+        public static let pengu = Image(.gmPengu)
```

Comment:
```suggestion
        public static let pengu = Image(.gmPengu)
```

there is should not be any pengu, clean and simple icon to represent what you want to do